consensus/scp/src/node.rs

```diff
@@ -44,10 +39,6 @@ pub struct Node<V: Value, ValidationError: Display> {
     /// Application-specific function for combining multiple values. Must be deterministic.
     combine_fn: CombineFn<V>,
 
-    /// Hashes of messages we've already processed.
-    /// (We store hashes instead of message content to reduce memory footprint.)
-    pub seen_msg_hashes: LruCache<Hash, ()>,
```

Comment:
Slot ignores duplicate or lesser messages on a per-node basis, which I think is simpler and more effective than maintaining a cache of messages here.